Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -365,14 +361,15 @@ public void Initialize(AppPlatform platform, AppSecret appSecret)
         _publishOnUpload = true;
         _overwriteDraftVersion = true;
 
-        if (_app.Removed)
-        {
-            Config.UnlinkApp(platform);
-        }
-
         try
         {
-            _appName = _app.Name;
+            App appInfo = Core.Api.GetAppInfo(appSecret);

```

Comment:
Use `var` here instead of `App` as type.

Reply:
done